package/gluon-web-admin/luasrc/lib/gluon/config-mode/model/admin/remote.lua

```diff
@@ -76,29 +75,12 @@ function pw2.cfgvalue()
 end
 
 local function set_password(password)
-	local inr, inw = unistd.pipe()
-	local pid = unistd.fork()
-
-	if pid < 0 then
-		return false
-	elseif pid == 0 then
-		unistd.close(inw)
-
-		local null = fcntl.open('/dev/null', fcntl.O_WRONLY)
-		unistd.dup2(null, unistd.STDOUT_FILENO)
-		unistd.dup2(null, unistd.STDERR_FILENO)
-		if null > 2 then
-			unistd.close(null)
-		end
-
-		unistd.dup2(inr, unistd.STDIN_FILENO)
-		unistd.close(inr)
-
-		unistd.execp('passwd', {[0] = 'passwd'})
-		os.exit(127)
-	end
+	local policies = {[unistd.STDIN_FILENO]=util.subprocess.PIPE,
+			  [unistd.STDOUT_FILENO]=util.subprocess.DEVNULL,
+			  [unistd.STDERR_FILENO]=util.subprocess.DEVNULL}
```

Comment:
here are some spacing issues left.

Reply:
Will fix them, when @neoraider has decided, what the keys are going to be.
https://github.com/freifunk-gluon/gluon/pull/2244#discussion_r664472959

Reply:
Sorry @2tata, thought this would happen faster. No need for you to wait as well ^^
resolved in 939720d